bundles/org.openhab.binding.wemo/README.md

```diff
@@ -24,6 +24,7 @@ In the thing file, this looks e.g. like
 ```
 wemo:socket:Switch1 [udn="Socket-1_0-221242K11xxxxx"]
 ```
+If for some reason the device IP address cannot be retrieved, it can be configured as an optional parameter, see examples below.
```

Comment:
If it **can** be retrieved, will setting the parameter have any effect? Would it for example skip some UPnP stuff and come online faster, or will it just be ignored? Any impact on reliability?

Reply:
No impact at all, this is just for text file configs. If it is set, the binding will not try to retrieve it again.

Reply:
To make it a bit more clear, the IP address is used for polling and command handling only. GENA events snd discovery use UPnP. Setting IP address will not let the thing go online faster, as this is depending on the UPnP registration.